src/web/nextui/src/app/components/InfoModal.css

```diff
@@ -0,0 +1,72 @@
+.info-modal {
+  outline: none;
+}
+
+.info-modal-content {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  max-width: 350px;
+  width: 90%;
+  max-height: 90vh;
+  overflow-y: auto;
+  background-color: var(--background-color);
+  color: var(--text-color);
+  border-radius: 8px;
+  box-shadow: 0 4px 20px rgba(0, 0, 0, 0.15);
+  transition: all 0.3s ease;
+  padding: 24px;
+}
+
+.info-modal-title {
+  margin-bottom: 8px;
+  font-size: 1.25rem;
```

Comment:
use `gutterBottom` or `mb` props instead of manually styling here

ideally, use the correct `variant` which supplies the size (it's ok if you have to override, but I'd double check that this is actually necessary)